src/components/panes/LocationPane.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import PaneBase from './PaneBase';
+import { retrieveLocation, updateLocation, deleteLocation,
+    setPendingLocation, clearPendingLocation } from '../../actions/location';
+import { getListItemById } from '../../utils/store';
+
+
+@connect(state => state)
+export default class LocationPane extends PaneBase {
+    componentDidMount() {
+        let locId = this.getParam(0);
+        let locItem = getListItemById(this.props.locations.locationList, locId);
+
+        if (locItem) {
+            this.props.dispatch(setPendingLocation(locItem.data));
+        }
+        else {
+            this.props.dispatch(retrieveLocation(locId));
+        }
+    }
+
+    getRenderData() {
+        let locId = this.getParam(0);
+
+        return {
+            locItem: getListItemById(this.props.locations.locationList, locId),
+        }
+    }
+
+    getPaneTitle(data) {
+        if (data.locItem) {
+            return data.locItem.data.title;
+        }
+        else {
+            return null;
+        }
+    }
+
+    renderPaneContent(data) {
+        if (data.locItem) {
+            console.log(data.locItem)
+            // TODO: Render form
+            return (
+                <div>
+                    <p>{ data.locItem.data.info_text }</p>
+                    <a onClick={() => {
+                        this.onLocationEdit(data.locItem.data)
+                    }}>Edit</a>
+                </div>
+            );
+        }
+        else {
+            // TODO: Show loading indicator?
+            return null;
+        }
+    }
+
+    onLocationEdit(loc) {
```

Comment:
Instead of passing the location to `onLocationEdit()`, you can retrieve the current location ID using `this.getParam(0)` (it's the first param to the pane in the URL/view store).